src/components/edit/overview/items-list.js

```diff
@@ -64,13 +65,16 @@ const AggregateDataExchangeCard = React.memo(({ ade }) => {
     }
 
     const [deleteConfirmationOpen, setDeleteConfirmationOpen] = useState(false)
+    const [sharingSettingsOpen, setSharingSettingsOpen] = useState(false)
     const closeDeleteConfirmation = useCallback(
         () => setDeleteConfirmationOpen(false),
         [setDeleteConfirmationOpen]
     )
 
+    const canShareExchange = ade.access.write && canAddExchange
```

Comment:
It might be worth using optional chaining here (e.g. `ade.access?.write` in case we don't have access information). That shouldn't happen, but sometimes the api request can behave unexpectedly (e.g. https://dhis2.atlassian.net/jira/software/c/projects/DHIS2/issues/DHIS2-17391) and we don't get back everything

Reply:
Surely, thanks Tom for flagging!